### DISCUSSION TITLE
RedHat SSO upcoming version will be based on which version of Keycloak ?

### DISCUSSION THREAD

**Discussion by somesh2012** [keycloak/keycloak#22601] [Archive]
Hi everyone,

Can anyone have any clue that the upcoming version of RedHat SSO will be based on which Keycloak version?
As the current version is RH SSO 7.6 which is based on Keycloak 18.0.8. I am curious about the upcoming version. When it will be going to release and based on which Keycloak version.
Please help me with your answers.

Thanks,
Somesh

**Comment by jonkoops** [comment]
We expect that the upcoming release of Red Hat SSO will be based on Keycloak 22. There is currently no ETA on when we will do the release.

**Reply by somesh2012** [reply]
Thanks for the reply. Really helpful.
